docs/src/less/ds6/index.less

```diff
@@ -14,6 +18,15 @@ h3 {
     .ds6-typography-headline-3;
 }
 
+.demo {
+    background-color: white;
+    border: 1px solid #eee;
```

Comment:
Same as above for #eee and white? Shall we use variables for consistency?

Reply:
I don't think these colours are in the DSL palette. Although white probably should be. Remember also this is just the CSS for the docs page, these aren't the skin src files :-)